mscore/prefsdialog.cpp

```diff
@@ -235,6 +236,7 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
       advancedWidget->loadPreferences();
       connect(advancedSearch, &QLineEdit::textChanged, this, &PreferenceDialog::filterAdvancedPreferences);
       connect(resetPreference, &QPushButton::clicked, this, &PreferenceDialog::resetAdvancedPreferenceToDefault);
+      connect(this, SIGNAL(preferencesChanged()), mscore->timeline(), SLOT(updateTimelineTheme()));
```

Comment:
Better use the new `connect` style just like the two lines above this line. The new style can check if the signal and slot exist so it can expose problems better than the old style (in case you change the names of signal and slot).